src/components/Admin/Configuration.vue

```diff
@@ -29,23 +29,23 @@
         <input
           type="submit"
           class="button-secondary margin-top-xl"
-          value="Get Project ID from Kontent"
+          value="Get Environment ID from Kontent"
         />
       </form>
     </section>
     <section class="section-secondary">
       <h2 class="headline-medium">Set A Project ID Manually</h2>
```

Comment:
You are right. However, there is still a difference where I think there should be project instead of environment(like sample project). I have tried to change what made sense to me now :)